pkg/volume/util/error.go

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Use this package instead https://github.com/kubernetes/cloud-provider/tree/master/volume/errors